.github/workflows/windows-tests.yml

```diff
@@ -67,7 +67,7 @@ jobs:
 
       - name: Upload DLL and PDB files as build artifacts
         if: ${{ github.event_name == 'push' }}
-        uses: actions/upload-artifact@v3
+        uses: actions/upload-artifact@v4
```

Comment:
This is the only action that had notable changes between v3 and v4 (see: [MIGRATION.md](https://github.com/actions/upload-artifact/blob/main/docs/MIGRATION.md), but I don't believe any of those changes applied to our usage.